language/libs/oslib.rd

```diff
@@ -108,12 +109,16 @@ items in linux {
 }
 
 items in redhat_family {
-  @cfengine_name="(redhat.!centos.!fedora.!oracle_linux.!scientific_linux.!amazon_linux)"
+  # Ugly but needed as very old rhel do not have the rhel class
+  @cfengine_name="rhel|(redhat.!centos.!fedora.!oracle.!almalinux.!rocky.!scientific_linux.!amazon_linux)"
   rhel,
   centos,
   fedora,
-  oracle_linux,
+  oracle,
+  almalinux,
+  rocky,
   scientific_linux,
+  @cfengine_name=["amzn", "amazon_linux"]
   amazon_linux,
```

Comment:
shouldn't it be amazon ?

Reply:
maybe, I just left it as it was but it would probably be more consistent

Reply:
but `if amazon {}` may be a little ambiguous as it does not design a linux distribution directly.